src/domain/repositories/ProgramsRepository.ts

```diff
@@ -0,0 +1,20 @@
+import { Async } from "domain/entities/Async";
+import { Id } from "domain/entities/Base";
+import { Pair } from "scripts/common";
+
+export interface ProgramsRepository {
+    closePatients(options: ClosePatientsOptions): Async<void>;
+}
+
+export interface ClosePatientsOptions {
+    orgUnitsIds?: Id[];
+    startDate?: string;
+    endDate?: string;
+    programId: Id;
+    programStagesIds: Id[];
+    closureProgramId: Id;
+    timeOfReference: string;
+    pairsDeValue: Pair[];
+    comments?: Pair;
+    post: boolean;
+}
```

Comment:
Note: We need entities if they are to be used in the domain (other entities, use cases, repos definition). Interfaces only used inside data repositories don't need (and shouldn't be) entities, the interfaces would live in a file inside `data/`.

Reply:
So you mean to declare the interface in data/repo and import it on domain/repo and domain/usecase? I think I'm not getting it